js/app/modules/card/knowledgeDocument.js

```diff
@@ -219,6 +221,35 @@ const KnowledgeDocument = new Module.Class({
                     height_request: this.MIN_CONTENT_HEIGHT,
                     visible: true,
                 });
+
+                this.content_view.connect('handle-external-link', (view, uri) => {
+                    if (GLib.uri_parse_scheme (uri) !== 'file')
+                        return false;
+
+                    let mediabin = new EosKnowledgePrivate.MediaBin({
+                        audioMode: true,
+                        uri: uri,
+                    });
+                    mediabin.show();
+
+                    let lightbox = new Lightbox.Lightbox({
```

Comment:
Right, once the audio files are in the shard we will be able to use a MediaLightBox

Th other option would be to pu the MediaBin in a popover but there is no easy way to get the link position from a PDFView (at least I havent found)